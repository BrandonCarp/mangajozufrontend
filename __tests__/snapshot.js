import { render } from "@testing-library/react";
import RootLayout from "@/app/layout";

it("renders homepage unchanged", () => {
  const { container } = render(<RootLayout />);
  expect(container).toMatchSnapshot();
});
