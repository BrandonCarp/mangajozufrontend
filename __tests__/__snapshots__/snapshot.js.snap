// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class=""
  >
    <nav
      class="flex bg-gray-700 text-gray-300 items-center  md:pr-10 h-[48px] "
      data-testid="main-nav"
      role="navigation"
    >
      <div
        class="flex items-center"
      >
        <nav
          class="w-full block md:hidden z-10"
          role="navigation"
        >
          <div
            data-headlessui-state=""
          >
            <button
              aria-expanded="false"
              class="flex items-center h-[48px]  p-3 hover:bg-gray-800 hover:text-white focus:bg-gray-800 active:bg-gray-800 focus:text-white border-none"
              data-headlessui-state=""
              id="headlessui-popover-button-:r0:"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="h-7 w-7"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
            </button>
          </div>
          <div
            style="position: fixed; top: 1px; left: 1px; width: 1px; height: 0px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border-width: 0px; display: none;"
          />
        </nav>
        <a
          class="  md:ml-10  text-purple-300 hover:text-white flex"
          href="/"
        >
          <svg
            aria-hidden="true"
            class="w-6"
            data-slot="icon"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M12 6.042A8.967 8.967 0 0 0 6 3.75c-1.052 0-2.062.18-3 .512v14.25A8.987 8.987 0 0 1 6 18c2.305 0 4.408.867 6 2.292m0-14.25a8.966 8.966 0 0 1 6-2.292c1.052 0 2.062.18 3 .512v14.25A8.987 8.987 0 0 0 18 18a8.967 8.967 0 0 0-6 2.292m0-14.25v14.25"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
          <p
            class="ml-1"
          >
            mangajozu
          </p>
        </a>
        <div
          class="w-full hidden md:block z-10"
          data-headlessui-state=""
        >
          <button
            aria-expanded="false"
            class="flex items-center h-[48px] p-3 hover:bg-gray-800 hover:text-white focus:bg-gray-800 focus:text-white border-none"
            data-headlessui-state=""
            id="headlessui-popover-button-:r5:"
            type="button"
          >
            <p
              class="pr-1"
            >
              Browse
            </p>
            <svg
              aria-hidden="true"
              class="w-6 h-6"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m19.5 8.25-7.5 7.5-7.5-7.5"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </button>
        </div>
        <div
          style="position: fixed; top: 1px; left: 1px; width: 1px; height: 0px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border-width: 0px; display: none;"
        />
      </div>
      <div
        class="ml-auto"
      >
        <div
          class="flex md:ml-5"
        >
          <a
            class="flex h-[48px] items-center justify-center gap-2  p-3 text-sm font-medium md:flex-none md:justify-start md:p-2 md:px-3 hover:bg-gray-800 hover:text-white"
            href="/search"
          >
            <svg
              aria-hidden="true"
              class="w-6 h-6"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m21 21-5.197-5.197m0 0A7.5 7.5 0 1 0 5.196 5.196a7.5 7.5 0 0 0 10.607 10.607Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </a>
          <a
            class="flex h-[48px] items-center justify-center gap-2  p-3 text-sm font-medium md:flex-none md:justify-start md:p-2 md:px-3 hover:bg-gray-800 hover:text-white hidden md:block md:pt-3"
            href="/favorites"
          >
            <svg
              aria-hidden="true"
              class="w-6 h-6"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M17.593 3.322c1.1.128 1.907 1.077 1.907 2.185V21L12 17.25 4.5 21V5.507c0-1.108.806-2.057 1.907-2.185a48.507 48.507 0 0 1 11.186 0Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </a>
          <a
            class="flex h-[48px] items-center justify-center gap-2  p-3 text-sm font-medium md:flex-none md:justify-start md:p-2 md:px-3 hover:bg-gray-800 hover:text-white"
            href="/profile"
          >
            <svg
              aria-hidden="true"
              class="w-6 h-6"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M15.75 6a3.75 3.75 0 1 1-7.5 0 3.75 3.75 0 0 1 7.5 0ZM4.501 20.118a7.5 7.5 0 0 1 14.998 0A17.933 17.933 0 0 1 12 21.75c-2.676 0-5.216-.584-7.499-1.632Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </a>
          <a
            class="flex h-[48px] items-center justify-center gap-2  p-3 text-sm font-medium md:flex-none md:justify-start md:p-2 md:px-3 hover:bg-gray-800 hover:text-white"
            href="/chat"
          >
            <svg
              aria-hidden="true"
              class="w-6 h-6"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M2.25 12.76c0 1.6 1.123 2.994 2.707 3.227 1.068.157 2.148.279 3.238.364.466.037.893.281 1.153.671L12 21l2.652-3.978c.26-.39.687-.634 1.153-.67 1.09-.086 2.17-.208 3.238-.365 1.584-.233 2.707-1.626 2.707-3.228V6.741c0-1.602-1.123-2.995-2.707-3.228A48.394 48.394 0 0 0 12 3c-2.392 0-4.744.175-7.043.513C3.373 3.746 2.25 5.14 2.25 6.741v6.018Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </a>
        </div>
      </div>
    </nav>
    <div
      class="relative w-screen h-screen overflow-hidden"
      data-testid="main-carousel"
    >
      <div
        class="absolute top-0 left-0 w-full h-full transition-opacity duration-1000 opacity-100"
      >
        <img
          alt="carousel-item"
          class="w-full h-full"
          data-nimg="1"
          decoding="async"
          height="40"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
          style="color: transparent; width: 100%; height: auto;"
          width="40"
        />
      </div>
      <div
        class="absolute top-0 left-0 w-full h-full transition-opacity duration-1000 opacity-0"
      >
        <img
          alt="carousel-item"
          class="w-full h-full"
          data-nimg="1"
          decoding="async"
          height="40"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
          style="color: transparent; width: 100%; height: auto;"
          width="40"
        />
      </div>
      <div
        class="absolute top-0 left-0 w-full h-full transition-opacity duration-1000 opacity-0"
      >
        <img
          alt="carousel-item"
          class="w-full h-full"
          data-nimg="1"
          decoding="async"
          height="40"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 640w, /_next/image?url=%2Fimg.jpg&w=750&q=75 750w, /_next/image?url=%2Fimg.jpg&w=828&q=75 828w, /_next/image?url=%2Fimg.jpg&w=1080&q=75 1080w, /_next/image?url=%2Fimg.jpg&w=1200&q=75 1200w, /_next/image?url=%2Fimg.jpg&w=1920&q=75 1920w, /_next/image?url=%2Fimg.jpg&w=2048&q=75 2048w, /_next/image?url=%2Fimg.jpg&w=3840&q=75 3840w"
          style="color: transparent; width: 100%; height: auto;"
          width="40"
        />
      </div>
    </div>
    <p
      class="text-white"
    >
      Footer Here
    </p>
  </div>
</div>
`;
